pandas_gbq/gbq.py

```diff
@@ -551,16 +551,20 @@ def run_query(self, query, **kwargs):
 
     def load_data(
             self, dataframe, dataset_id, table_id, chunksize=None,
-            schema=None):
+            schema=None, progress_bar=False):
         from pandas_gbq import load
 
         total_rows = len(dataframe)
         logger.info("\n\n")
 
         try:
-            for remaining_rows in load.load_chunks(
-                    self.client, dataframe, dataset_id, table_id,
-                    chunksize=chunksize, schema=schema):
+            chunks = load.load_chunks(self.client, dataframe, dataset_id,
+                                      table_id, chunksize=chunksize,
+                                      schema=schema)
+            if progress_bar:
+                from tqdm import tqdm
+                chunks = tqdm(chunks)
```

Comment:
In many cases there will be just one chunk, even for quite large dataframes. To be a useful progress indicator, this change should manually poll to see if the load job is complete as is done for query jobs:

https://github.com/pydata/pandas-gbq/blob/8f49ec3f134cc0a85f541116c42647c515fdc7e6/pandas_gbq/gbq.py#L504-L522

Reply:
Well, tqdm can to anything in this case. It's more useful for more number of chunks. The percentage done is based on the proportion of of chunks completed wrt the total number of chunks.

Reply:
I'd caution against this change in that case. Load jobs are a limited resource (1,000 per day), so I'd rather encourage people to use fewer load jobs than more. Chunks should only be used when there is extreme memory pressure and a dataframe does not fit into memory if we try to serialize to CSV before upload.